### PR TITLE
Ports over some Traits from Hippie Station!

### PR DESCRIPTION
**Pull request by fluffe9911**
🆑 fluffe9911
add: Monochrome, Brown and Grey, Inverted Red and Blue Colors and Waddle Trait! 
/🆑
I regret nothing!

### CODE REVIEW

**Comment by Time-Green** [comment]
We already have monochromy, and we also already have the waddle component, it’s just unused
I think you might want to remove those dupes

File: yogstation/code/datums/traits/neutral.dm [hunk 1]
```diff
@@ -0,0 +1,76 @@
+/* Yogstation Neutral Traits */
+/datum/quirk/monochromatic
```
Comment (Amelia0010) [inline yogstation/code/datums/traits/neutral.dm]: this is already a thing though https://github.com/yogstation13/Yogstation-TG/blob/c24df24f82d1cdb444d2b18ffa43223e1fdcef89/code/datums/traits/neutral.dm#L70

**Review by Amelia0010 — CHANGES_REQUESTED** [review]

**Comment by fluffe9911** [comment]
The monochrome was just me not paying enough attention to traits and waddling was actually never enabled until now and theres a bit extra to it also made waddling cost a point due to it being a bit memey

**Comment by Time-Green** [comment]
I said the component was already added but unused, not the quirk, thats new
https://github.com/yogstation13/Yogstation-TG/blob/76b6bca91b5577918132a8cf5def6834978e3786/code/datums/components/waddling.dm
The already existing waddle component is exactly the same, safe for a few sanity checks and additional variables

File: yogstation/code/datums/traits/neutral.dm [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+/datum/quirk/inverted_vision/remove()
+	var/mob/living/carbon/human/H = quirk_holder
+	if(H)
+		H.remove_client_colour(/datum/client_colour/inverted)
+
+/datum/quirk/waddle
+	name = "Waddle"
+	desc = "You have an extra bounce in your step."
+	value = 1
```
Comment (Amelia0010) [inline yogstation/code/datums/traits/neutral.dm:39]: honestly, having the waddle component means it's harder to change identity, whether it's via changeling transform or via chameleon kit, so maybe it should be 0 points or negative. *Or maybe a genetics mutation*

**Comment by 1fbff5f83b23d39d38b1dfcb4cac8d9b** [comment]
fluffe's computer broke

**Comment by Amelia0010** [comment]
oof

**Pull request closed without merging**